gax/src/main/java/com/google/api/gax/rpc/UnaryCallSettings.java

```diff
@@ -96,9 +96,36 @@ public String toString() {
         .toString();
   }
 
+  @Override
+  public int hashCode() {
```

Comment:
Are equals/hascode manually written? Not a big deal but it is better to have equals/hashcode implementation in gax consistent with the others.